packages/cli/src/commands/generate/graphql-client-code/generate_graphql_client_code_command.ts

```diff
@@ -89,20 +94,38 @@ export class GenerateGraphqlClientCodeCommand
   handler = async (
     args: ArgumentsCamelCase<GenerateGraphqlClientCodeCommandOptions>
   ): Promise<void> => {
-    const backendIdentifier =
-      await this.backendIdentifierResolver.resolveDeployedBackendIdentifier(
-        args
-      );
-    const out = this.getOutDir(args);
-    const format = args.format ?? GenerateApiCodeFormat.GRAPHQL_CODEGEN;
-    const formatParams = this.formatParamBuilders[format](args);
+    try {
+      const backendIdentifier =
+        await this.backendIdentifierResolver.resolveDeployedBackendIdentifier(
+          args
+        );
+      const out = this.getOutDir(args);
+      const format = args.format ?? GenerateApiCodeFormat.GRAPHQL_CODEGEN;
+      const formatParams = this.formatParamBuilders[format](args);
 
-    const result = await this.generateApiCodeAdapter.invokeGenerateApiCode({
-      ...backendIdentifier,
-      ...formatParams,
-    } as unknown as InvokeGenerateApiCodeProps);
+      const result = await this.generateApiCodeAdapter.invokeGenerateApiCode({
+        ...backendIdentifier,
+        ...formatParams,
+      } as unknown as InvokeGenerateApiCodeProps);
 
-    await result.writeToDirectory(out);
+      await result.writeToDirectory(out);
+    } catch (error) {
+      if (
+        error instanceof BackendOutputClientError &&
```

Comment:
As per the lint, don't use instanceof as it might not work if there are multiple versions of DeployedBackendClient package in the node_modules.